### ISSUE TITLE
Profile archives have "Link missing a text alternative" error on email. icon in SiteImprove Accessibility

### ISSUE THREAD

**Issue by troyrutter** [iastate/iastate22-wordpress#56]
The icon for email on a profile archive page gets a "Link missing a text alternative" error from SiteImprove.

An example of the div is attached

![Image](https://github.com/user-attachments/assets/c47efbd6-dad2-46ca-990e-35d6697f88d0)

The suggested remedy is to add an aria label to the href.
